diff --git a/config.def.h b/config.def.h
index 6f05dce..2eb7750 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+static char *font = "DMMono Nerd Font:pixelsize=18:antialias=true:autohint=true";
+static int borderpx = 9;

 /*
  * What program is execed by st depends of these precedence rules:
