diff --git a/config.def.h b/config.def.h
index 3eabc86..10f39ec 100644
--- a/config.def.h
+++ b/config.def.h
@@ -191,9 +191,9 @@ static Shortcut shortcuts[] = {
 	{ ControlMask,          XK_Print,       toggleprinter,  {.i =  0} },
 	{ ShiftMask,            XK_Print,       printscreen,    {.i =  0} },
 	{ XK_ANY_MOD,           XK_Print,       printsel,       {.i =  0} },
-	{ TERMMOD,              XK_Prior,       zoom,           {.f = +1} },
-	{ TERMMOD,              XK_Next,        zoom,           {.f = -1} },
-	{ TERMMOD,              XK_Home,        zoomreset,      {.f =  0} },
+	{ TERMMOD,              XK_plus,        zoom,           {.f = +3} }, // ctrl +
+	{ TERMMOD,              XK_underscore,  zoom,           {.f = -3} }, // ctrl -
+	{ TERMMOD,              XK_parenright,  zoomreset,      {.f =  0} }, // ctrl 0
 	{ TERMMOD,              XK_C,           clipcopy,       {.i =  0} },
 	{ TERMMOD,              XK_V,           clippaste,      {.i =  0} },
 	{ TERMMOD,              XK_Y,           selpaste,       {.i =  0} },
