diff --git a/defaults.go b/defaults.go
index bd9868a..40e1eb1 100644
--- a/defaults.go
+++ b/defaults.go
@@ -264,7 +264,7 @@ var defaults = Config{
 			LoadAvgValue:     5,
 			LoadThresholdBad: 1.0,
 
-			NixShellFg: 00,
+			NixShellFg: 15,
 			NixShellBg: 69, // a light blue
 
 			DurationFg: 250,
diff --git a/segment-nix-shell.go b/segment-nix-shell.go
index 00f8c45..af2ad1c 100644
--- a/segment-nix-shell.go
+++ b/segment-nix-shell.go
@@ -13,7 +13,7 @@ func segmentNixShell(p *powerline) []pwl.Segment {
 	}
 	return []pwl.Segment{{
 		Name:       "nix-shell",
-		Content:    nixShell,
+		Content:    "\uf313",
 		Foreground: p.theme.NixShellFg,
 		Background: p.theme.NixShellBg,
 	}}
diff --git a/themes/default.json b/themes/default.json
index 39434bb..3c00e57 100644
--- a/themes/default.json
+++ b/themes/default.json
@@ -69,7 +69,7 @@
   "LoadHighBg": 161,
   "LoadAvgValue": 5,
   "LoadThresholdBad": 1.0,
-  "NixShellFg": 0,
+  "NixShellFg": 15,
   "NixShellBg": 69,
   "DurationFg": 250,
   "DurationBg": 237,
